src/abstraction.jl

```diff
@@ -1,6 +1,8 @@
 module Abstraction
 
 using LinearAlgebra
+using PyPlot
```

Comment:
This is causing CI to fail: https://travis-ci.com/github/blegat/Dionysos.jl/jobs/367052963#L293-L337

Reply:
ah oops, j'ai oublié de l'enlever! Bien vu ;)